### PR TITLE
Vn development merge into unstable

### LINKED ISSUE #1956: multi spawned statics in milBuildings
**Describe the bug**
When multiple locations of type `outpost` or `airport` have overlap with its `marker size * 1.2` it can spawn multiple statics in the designated static spawn positions.
(scalable to N locations overlapping over the mil building in question).

**To Reproduce**
Steps to reproduce the bug (If you can't reproduce it, just give details of what happened beforehand. Anything you think could be relevant):
spawn in a locations with a mil building being covered by multiple airport or outpost markers.

**Expected behaviour**
singular static in each spawn location spawned

**Screenshots**

**Environment (please complete the following information):**
Version:  2.4.1

Did you make any changes to the code and if so, which changes: No

Map (specify if Altis Blufor): 

* [x] LAN Hosted
* [x] Private dedicated server
* [x] Private dedicated server with Headless client(s)
* [ ] Community Server 1
* [ ] Community Server 2
* [ ] Community Test Server
* [ ] Community Test Server 2

Modset: Any
    
Time the bug occurred (Please leave in GMT+0): N/A
If **Not** on a community server, please leave the time zone of your client for LAN, or the server for dedicated: 

**Additional context**

Note we could potentially solve this with registre on the buildings of which locations are spawning it in, and only create a static if no other location has spawned it yet.
we may also consider blocking spawning statics in markers that is not the markes spawning stuff. (locations outside markers would still spawn.

If you were LAN hosting, please attach your client .RPT file from when the bug occurred, If on a private dedicated server, please attach the server's .RPT from when the bug occurred.

### PR DESCRIPTION
**Pull request by Bob-Murphy**
## What type of PR is this.
This PR merges the vn-development branch into unstable to combine the branches

### What have you changed and why?
vn-development was created on the NDA repo first and it contains the complete adaptation of Antistasi to the CLN map and the rest of the S.O.G. Prairie Fire CDLC.
for a list of changes see
https://github.com/official-antistasi-community/A3-Antistasi/releases/tag/2.4.1.VN.01
and
https://github.com/official-antistasi-community/A3-Antistasi/releases/tag/2.4.1.VN.02
as well as all merged PRs with the label "vn-development"

### What needs to be done?
This PR will have a ton of merge conflicts which will have to be resolved and the final product will need some thorough testing.

********************************************************
Notes:
closes #1956 as it's already fixed on this branch

### CODE REVIEW

**Comment by Bob-Murphy** [comment]
Technically the PR should be ready to merge.
Reason being:
- we merged unstable into the vn-development branch 14 days ago
- in the last 14 there have only been 2 merged commits to unstable which are
- --> a change in the bug report templates https://github.com/official-antistasi-community/A3-Antistasi/commit/a2f5c268a49ad5e78038e163f67021151388d48c
- --> a typo fix within a message https://github.com/official-antistasi-community/A3-Antistasi/commit/0465d0c3498ff61737abc694067d8b35cdb4734c
- the mission from vn-development has been tested, is published and running

**Comment by Bob-Murphy** [comment]
Things to do after merge:
- [ ] set version number in stringtable

File: A3-Antistasi/functions/Base/fn_setPlaneLoadout.sqf [hunk 1]
```diff
@@ -64,11 +64,23 @@ if (_type == "CAS") then
         case "UK3CB_TKA_B_Su25SM_CAS";
         case "UK3CB_CW_SOV_O_LATE_Su25SM_CAS":
         {
-            _loadout = ["rhs_mag_kh29D","rhs_mag_kh29D","rhs_mag_kh25MTP","rhs_mag_kh25MTP","rhs_mag_kh25MTP","rhs_mag_kh25MTP","rhs_mag_b8m1_s8kom","rhs_mag_b8m1_s8kom","rhs_mag_R60M","rhs_mag_R60M","rhs_ASO2_CMFlare_Chaff_Magazine_x4"];
+            _loadout = ["vn_bomb_f4_out_750_blu1b_fb_mag_x2","vn_bomb_f4_out_750_blu1b_fb_mag_x2","vn_rocket_ffar_f4_lau3_m229_he_x57","vn_rocket_ffar_f4_lau3_m229_he_x57","vn_bomb_f4_in_2000_gbu8_he_mag_x1","vn_missile_f4_lau7_aim9e_mag_x2","vn_missile_f4_lau7_aim9e_mag_x2","vn_missile_aim7e2_mag_x1","vn_missile_aim7e2_mag_x1","vn_missile_aim7e2_mag_x1","vn_missile_aim7e2_mag_x1"];
```
Comment (Lazejun) [inline A3-Antistasi/functions/Base/fn_setPlaneLoadout.sqf:67]:
```suggestion
            _loadout = ["rhs_mag_kh29D","rhs_mag_kh29D","rhs_mag_kh25MTP","rhs_mag_kh25MTP","rhs_mag_kh25MTP","rhs_mag_kh25MTP","rhs_mag_b8m1_s8kom","rhs_mag_b8m1_s8kom","rhs_mag_R60M","rhs_mag_R60M","rhs_ASO2_CMFlare_Chaff_Magazine_x4"];
```

**Comment by Lazejun** [comment]
Besides that we might wanna Clean up the .bak Files

**Review by HakonRydland — APPROVED** [review]
all changes in this has been tested and reviewed as part of vn-dev or pr to it. everything ***should*** therefore be good

**Pull request merged**